docs/docs/core/helper-plugin/hooks/use-fetch-client.mdx

```diff
@@ -39,10 +39,6 @@ const Component = () => {
 }
 ```
 
-:::tip
-Remember to use a relative path for your requestURL, following this format `/{yourRelativePath}`
-:::
```

Comment:
We should add something about what the expected url style is e.g. I believe it's only URLs with a protocol or a relative url not `www.google.com`?

Reply:
let me know if it is ok